JustSaying.Models/Message.cs

```diff
@@ -20,8 +19,6 @@ protected Message()
         public string Conversation { get; set; }
         public string ReceiptHandle { get; set; }
         public Uri QueueUri { get; set; }
-        public int? DelaySeconds { get; set; }
-        public IDictionary<string, MessageAttributeValue> MessageAttributes { get; set; }
```

Comment:
Perhaps continue to populate these and add an obsolete attribute, then remove? (that said, the fix is easy)

Reply:
Not widely used, can be removed in a major version update as long as the fix is documented.